frontend/src/components/node/List.tsx

```diff
@@ -58,7 +59,7 @@ export default function NodeList() {
         {
           id: 'ready',
           label: t('translation|Ready'),
-          gridTemplate: 'minmax(150px, .3fr)',
+          gridTemplate: 'minmax(120px, .3fr)',
```

Comment:
maybe we could just delete this line?

Reply:
your suggestion actually looks cleaner, removed the line here